gdx-backend-jtransc/src/com/jtransc/media/limelibgdx/LimeInput.java

```diff
@@ -443,17 +449,9 @@ private static int convertKeyCode(int i) {
 
 	private boolean checkKeyArray(boolean[] array, int i) {
 		if (i < 0) {
```

Comment:
it's good!

Exception in input anyway not good, because we see many strange on different platforms =) but after finish with all, i fix it ( if remember =) )